libs/blocks/merch/merch.js

```diff
@@ -721,6 +721,21 @@ export async function buildCta(el, params) {
       reopenModal(cta);
     });
   }
+
+  // Adding aria-label for checkout-link using productFamily as placeholder key and card-heading
+  if (!cta.getAttribute('aria-label')) {
+    cta.onceSettled().finally(async () => {
+      const productName = cta.value[0]?.productArrangement?.productFamily;
+      const merchCard = cta.closest('merch-card');
+      if (productName) {
+        await replaceKey(productName, getConfig()).then((label) => {
+          const cardHeading = merchCard ? ` - ${merchCard.querySelector('.card-heading')?.textContent}` : '';
```

Comment:
as mentioned in JIRA,
if its a merch link we don't need info from merch-card, let's just use WCS response:
```
{{Card Text}} {{Product Family}} {{Customer Segment}}
e.g.
{{Card Text}} {{cta.value[0]?.productArrangement?.productFamily}} {{cta.value[0]?.customerSegment}}
```

you would need to avait for both placeholders, productFamily and customerSegment

Reply:
You mean, {{CTA text}}, right?